cangjie5.dict.yaml

```diff
@@ -79372,3 +79371,4 @@ encoder:
 𫜉	ywwlb
 𫐼	yybs
 𫑉	yysb
+𬌗	mhomr
```

Comment:
𬌗 is an Ext. E character and thus should be placed after all other characters.

Reply:
Note that it is OK to move 𬌗 here because `mhomr` is unique, so it has no observable behaivor changes.